proc_echo_mw.py

```diff
@@ -19,19 +12,20 @@
 logger = init_logging("info")
 t2 = symbols('t2')
 signal_pathway = {'ph1':1}
-freq_range= (-5e3,5e3)
+# shouldn't be part of the pull request
+freq_range= (-4e3,4e3)
 t_range = (0,0.083)
 fl = fl_mod()
 #}}}
 for filename,nodename,file_location,postproc in [
-        ('210609_TEMPOL_100mM_cap_probe_DNP','enhancement',
+        ('210614_TEMPOL_500uM_cap_probe_DNP','enhancement',
             'ODNP_NMR_comp/test_equipment','spincore_ODNP_v1'),
         ]:
     s = find_file(filename,exp_type=file_location,expno=nodename,
             postproc=postproc,lookup=lookup_table,fl=fl)
     myslice = s['t2':freq_range]
-    mysign = determine_sign(select_pathway(myslice,signal_pathway,mult_ph_dims=False))
-    enhancement,idx_maxpower = process_enhancement(s,searchstr = filename,
-            freq_range=freq_range, t_range=t_range,flip=True,sign=mysign,fl=fl)
+    mysign = select_pathway(myslice, signal_pathway).real.sum('t2').run(np.sign)
+    enhancement = process_enhancement(s,freq_range=freq_range, 
+            t_range=t_range,sgn=mysign,fl=fl)
     fl.show()
```

Comment:
these changes should not be part of this pull request!

Reply:
need to test alignment on actual experimental enhancement data to ensure it works as it should